module5/lessons/Week1/CoffeeChatKickoff.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Coffee Chat Kickoff
+---
+
+In Mod 4, you all did a great job with your first coffee chat! I know for some of you it was quite stressful, my first couple coffee chats definitely were! I hope you feel proud of yourself for breaking the ice and are now ready to keep this energy moving forward 🚀
+
+As we talked about in the CoffeeChatPD lesson there are many benefits of coffee chats including:
+
+**Real-world Context:** Coffee chats help bridge the gap between theoretical knowledge gained in class and real-world applications. Interviewees will share real world examples that illustrate how coding skills are used in actual projects and workflows.
+
+**Networking:** Engaging in coffee chats allows you to expand your professional network. Building connections with experienced professionals can lead to mentorship opportunities, potential job referrals, and a broader understanding of the industry landscape. But it’s key to note, the tone of the coffee chat should not be job seeking, even though there is a fair chance it might help lead to a job.
+
+**Interview Preparation:** While you are not the one interviewed in this context, coffee chats still develop your comfort level talking with an unknown professional and help the actual interviews feel less scary.
+
+## The Assignment
+
+The expectation for Mod 5 is that **you will have two more coffee chats by the end of Week 4**. 
+
+✅Deliverable✅: By the end of Week 4, send the LinkedIns of the two people you chatted with along with your answers to the following reflection questions:
+
+* How was your experience finding someone to chat with?
+
+* What are some of the takeaways from your conversations?
+
+* What do you want to do similarly in your next coffee chat?
+
+* What do you want to do differently in your next coffee chat?
+
+
+*If you haven't found two people to chat with by the end of Week 2, reach out to your instructors. In your message, include everyone you tried reaching out to who hasn't responded, there should be at least 8 folks!*

```

Comment:
8 feels like a lot for the end of week 2?  This is one person per day (they should do multiple outreaches at once of course)... but this is a lot of ghosting/rejection... could we maybe say 6?

Reply:
Although with time set in the lesson to find suitable chatters... 8 could be reasonable

Reply:
I'm totally good with 6. Wasn't sure exactly what the number is for, you've put in a solid effort on your own and it's not working. Need them to first put in solid effort before we assist.

Reply:
I just don't know how much ghosting they will get. When I talked with some mod 3/4 students at a state of Turing a little bit ago they were saying they haven't done any coffee chats and were not getting responses from people when they messaged, but I think with a thoughtful message 50% response feels reasonable. We will see!